users/views.py

```diff
@@ -1,3 +1,116 @@
-# from django.shortcuts import render
+from datetime import datetime
+from calendar import Calendar
+from users.models import User
+from django.contrib import messages
+from users.forms import MyUserCreationForm
+from django.shortcuts import redirect, render
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth import authenticate, login, logout
 
-# Create your views here.
+Calendar = Calendar(6)  # 6 => means that the week start with Sunday
```

Comment:
I think maybe my point wasn't clear
this is the overwrite I don't want to see
I meant something like:
```suggestion
cal = Calendar(6)  # 6 => means that the week start with Sunday
```

Reply:
ok